src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -218,6 +219,8 @@ py::object from_ov_any(const ov::Any& any) {
         return py::cast(any.as<ov::streams::Num>());
     } else if (any.is<ov::Affinity>()) {
         return py::cast(any.as<ov::Affinity>());
+    } else if (any.is<ov::WorkloadType>()) {
+        return py::cast(any.as<ov::WorkloadType>());
```

Comment:
Also need to add to py_object_to_any function.

Reply:
Done